examples/routing_service/routing_service_struct_array_transf/cpp/README.md

```diff
@@ -61,6 +61,17 @@ export RTI_LD_LIBRARY_PATH=$NDDSHOME/lib/<ARCH>
 where `<ARCH>` shall be replaced with the target architecture you used to build
 the example in the previous step.
 
+**Note:** If you wish to run the executable for a target architecture different
+than the host, you have to specify your architecture. That way the Routing
+Service script can use the specific target binary instead of using the standard
+host binary. This can be done by using the `CONNEXTDDS_ARCH` variable.
+
+```sh
+export CONNEXTDDS_ARCH=<ARCH>
```

Comment:
Mention a real architecture to be consistent. ``export CONNEXTDDS_ARCH=x64Linux3gcc5.4.0``.